### DISCUSSION TITLE
Problems with Logout Button (fake login)

### DISCUSSION THREAD

**Discussion by JanaS87** [neuefische/questions#721] [Web] [answered]
Hey everybody,

I have some problems with my fake login.
When I enter my login data, I´ll get this messages (Browser Firefox):
![Bildschirmfoto 2023-09-06 um 11 14 40](https://github.com/neuefische/questions/assets/136725332/560c0a4e-ca58-4655-b13e-ad60d9d76238)

Login works so far but my user greetings are empty. 
Perhaps I've forgotten to pass the right props somewhere?
In the console.log(), the user is: undefined.

My logout button unfortunately doesn't show any signs of life. The console.logs also don't respond to the button, so I must've made a mistake somewhere there.

Can someone please take a look and tell me where I went wrong? When I feed ChatGPT with the question and the information, it always suggests Context or Redux or something...

I don't have a database; everything runs through localStorageState().
I'd like to get this done by Friday, and I'm starting to feel the pressure = even more mistakes.

My current PR: [PR](https://github.com/JanaS87/capstone-project/pull/28)

Thanks so much in advance <3

**Comment by Kheirah** [comment, marked as answer]
In `_app.js` you forgot to pass down the props to your `LogoutButton` component 😉 
```js
{loggedIn && <LogoutButton loggedIn={loggedIn} handleLogout={handleLogout} />}
```

**Reply by JanaS87** [reply]
I am soooo blind 😄 

Thank you so much! 

Do you have any suggestions, what I can do with the "warnings" that I get?

**Reply by Kheirah** [reply]
Oh, I didn't inspect them. Ehrm, ... no I have no idea either 😁 
Maybe Google / ChatGPT can help with that.
